src/containers/results/results.tsx

```diff
@@ -48,8 +47,10 @@ export const Results: FC<Props> = ({ gainedScore, botsBusted, totalBots }) => {
             signature,
             callbackUrl: pages.home,
           });
+          setMergeRequested(false);
         } else {
           await router.push(pages.usernameSelect);
+          setMergeRequested(false);
```

Comment:
this should be above router.push, else this gets triggered after components unmounts and that's a leak

Reply:
This would not solve my comment since it would still be executed after

Reply:
Then we're back to the beginning and the double clicking would be back